multiqc/modules/sortmerna/sortmerna.py

```diff
@@ -0,0 +1,151 @@
+                s_name = os.path.basename(l.split(" ")[-1]).strip().split(".")[0]
+                self.sortmerna[s_name] = dict()
+            if "Results:" in l and not post_results_start:
+                post_results_start = True
+            if not post_results_start:
+                continue
+            if post_results_start and not post_database_start:
+                if "Total reads =" in l:
+                    m = re.search("\d+",l)
+                    if m:
+                        self.sortmerna[s_name]["total"] = int(m.group())
+                    else:
+                        err = True
+                elif "Total reads passing" in l:
+                    m = re.search("\d+",l)
+                    if m:
+                        self.sortmerna[s_name]["rRNA"] = int(m.group())
+                        self.sortmerna[s_name]["rRNA_pct"] = float(self.sortmerna[s_name]["rRNA"]) / float(self.sortmerna[s_name]["total"]) * 100
+                    else:
+                        err = True
+                elif "Total reads failing" in l:
+                    m = re.search("\d+",l)
+                    if m:
+                        self.sortmerna[s_name]["non_rRNA"] = int(m.group())
+                        self.sortmerna[s_name]["non_rRNA_pct"] = float(self.sortmerna[s_name]["non_rRNA"]) / float(self.sortmerna[s_name]["total"]) * 100
+                    else:
+                        err = True
+            if post_database_start:
+                if not l.strip():
+                    break
+                db_number = db_number + 1
+                m = re.search("    .*\t", l)
+                if m:
+                    db = m.group().strip()
+                    db = os.path.splitext(os.path.basename(db))[0]
+                    pct = float(re.search("\d+\.\d+%", l).group().replace("%",""))
+                    count = int(self.sortmerna[s_name]["total"]) * (pct / 100)
+                    self.sortmerna[s_name][db + "_pct"] = pct
+                    self.sortmerna[s_name][db + "_count"] = count
+                else:
+                    err = True
+            if "By database:" in l and not post_database_start:
+                post_database_start = True
+        if err:
+            log.warning("Error parsing data in: " + s_name)
+            self.sortmerna.pop(s_name, 'None')
+        s_name = None
+
+    def sortmerna_overall_barplot (self):
+        keys = OrderedDict()
+        keys["non_rRNA"] = { 'color': '#a6cee3', 'name': 'Other' }
+        keys["rRNA"] = { 'color': '#e31a1c', 'name': 'rRNA' }
+        pconfig = {
+            'id': 'SortMeRNA overall',
+            'title': 'SortMeRNA Other vs rRNA',
+            'ylab': 'Reads'
+        }
+        log.debug(keys)
+        return plots.bargraph.plot(self.sortmerna, keys, pconfig)
+
+    def sortmerna_detailed_barplot (self):
+        """ Make the HighCharts HTML to plot the sortmerna rates """
+
+        colors = ["#1f78b4", "#b2df8a", "#33a02c", "#fb9a99",
```

Comment:
Any reason for the non-default colours? Is this matching output from SortMeRNA or something?